app/src/main/java/com/microsoft/device/samples/dualscreenexperience/presentation/devmode/DevContentFragment.kt

```diff
@@ -34,25 +36,41 @@ class DevContentFragment : Fragment() {
         savedInstanceState: Bundle?
     ): View? {
         binding = FragmentDevContentBinding.inflate(inflater, container, false)
+        binding?.isLoading = true
+        binding?.isConnected = true
         return binding?.root
     }
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
+        setupNetworkObserver()
         setupWebView()
         setupObservers()
     }
 
-    private fun setupObservers() {
-        viewModel.webViewUrl.observe(
-            viewLifecycleOwner,
-            {
-                it?.let {
-                    binding?.devContentWebView?.loadUrl(it)
+    private fun setupNetworkObserver() {
+        NetworkConnectionLiveData(activity?.applicationContext).observe(viewLifecycleOwner) {
+            if (it) {
```

Comment:
Optional: It is easier to understand the code when we set a name to "it", something like "isConected".

Reply:
Updated, thanks:)